experiments/standalone/Snow/snowmip_simulation_gpu.jl

```diff
@@ -0,0 +1,452 @@
+import SciMLBase
+import ClimaTimeSteppers as CTS
+using CairoMakie
+import ClimaParams as CP
+import ClimaUtilities.TimeVaryingInputs: TimeVaryingInput
+import ClimaUtilities.OutputPathGenerator: generate_output_path
+import ClimaUtilities.ClimaArtifacts: @clima_artifact
+using ClimaCore
+using ClimaLand.Snow
+using ClimaLand.Domains
+using ClimaComms
+import ClimaLand
+import ClimaLand.Parameters as LP
+using DataFrames
+using NCDatasets
+using Dates
+using ClimaLand: PrescribedAtmosphere, PrescribedRadiativeFluxes
+using Thermodynamics
+using Statistics
+using Insolation
+using DelimitedFiles
+ClimaComms.@import_required_backends
+using ClimaUtilities.Utils: linear_interpolation
+
+using CSV, HTTP, Flux, BSON, StatsBase
+NeuralSnow = Base.get_extension(ClimaLand, :NeuralSnowExt).NeuralSnow;
+
+# Site-specific quantities
+# Error if no site argument is provided
+if length(ARGS) < 1
+    # @error("Please provide a site name as command line argument")
+    SITE_NAME = "cdp"
+else
+    SITE_NAME = ARGS[1]
+end
+climaland_dir = pkgdir(ClimaLand)
+
+FT = Float32
+context = ClimaComms.context()
+ClimaComms.init(context)
```

Comment:
maybe we could grab the device in the original snowmip simulation script, and if device == cpu make the plots, and if GPU, dont make them. Is it worth checking that the output is the same between the two runs?